README.MD

```diff
@@ -14,13 +14,7 @@ Total message size (bytes): Payload + 3
 
 #### Install Arduino Libraries
 
-Copy the `src\SerialProtocol` folder to:
-
-- Windows: `%USERPROFILE%\Victor\Documents\Arduino`
-- Linux: `~/sketchbook/libraries` or `/usr/share/arduino/libraries`
-- Mac: `Documents/Arduino/libraries`
-
-Note: You must restart the Arduino IDE if it was running before you copied the files.
+Download the Repository, Go to Sketch>Include Library>Add .zip Library> Select the Downloaded .zip File.
```

Comment:
can you please make this a bit more explicit by saying "Download the repository as zip"? Someone might get confused by it and just git clone it.

Also nitpick on this line. `>` follow by spaces in some cases but not in others (consistency)